lightning/checkpoints/checkpoints.go

```diff
@@ -329,8 +333,61 @@ type DestroyedTableCheckpoint struct {
 	MaxEngineID int32
 }
 
+type TaskCheckpoint struct {
+	TaskId       int64
+	SourceDir    string
+	Backend      string
+	ImporterAddr string
+	TidbAddr     string
```

Comment:
how about store the host and port in separate fields 🤷 

```suggestion
	TiDBHost     string
	TiDBPort     uint16
```